ui/main/src/app/modules/archives/archives.component.html

```diff
@@ -85,6 +127,9 @@
         <td translate [translateParams]="card.summary.parameters">
           {{ card.process + '.' + card.processVersion + '.' + card.summary.key}}
         </td>
+        <td translate>
+          {{ this.findServiceForProcess(card.process) }}
```

Comment:
Check if displayServiceFilter() as for column header

Reply:
Done.